specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/managedClusters.json

```diff
@@ -2985,6 +3080,61 @@
           "description": "The private link service connection description."
         }
       }
+    },
+    "PrivateLinkResourcesListResult": {
```

Comment:
for reviewers: we has only 1 item in the list currently, so pagination is not needed.